app/webpack/reactjs/types/modelTypes.ts

```diff
@@ -20,9 +20,44 @@ export interface Studio extends ActiveRecordModel {
 }
 
 export interface ArtPiece extends ActiveRecordModel {
-  slug: string;
+  title: string;
+  dimensions: string;
+  year: string;
+  price: string;
+}
+
+export interface JsonApiModel<T, R> {
```

Comment:
Our typescript verification story is really weak so I'm not certain all this is right the idea is there.